### ISSUE TITLE
NO DC msg, simple video coding bug and a question

### ISSUE THREAD

**Issue by FastHit** [visualboyadvance-m/visualboyadvance-m#852]
### Expected behavior
when i set the output module to simple to work just fine

### Actual behavior
the video offset itself out of bounds

### Steps to reproduce the behavior
just set the video to simple

### ROM details

Which games are affected by the issue: pokemon ROMS it's GBA.

Include screenshots of ROM Information (File > ROM Information):

![image](https://user-images.githubusercontent.com/54322185/121032968-7374b080-c7b4-11eb-9d0b-57e86f0e7756.png)

### Build details

Operating System: windows 10

VBA-M version: 2.1.4

### in short, and additional question

1. when i play on it(using pokemon fire red ROM or any other pokemon rom i have), there is "no DC" msg in the top right and it is quite annoying for me.
2. when i change to simple video driver the video bugs out and is offset way to the left.
3. is there an option to make the on screen messages smaller?(they are way too big)

here are some screenshots showing it
https://imgur.com/a/pHP3eI4

**Comment by Squall-Leonhart** [comment]
NO DC is coming from an external source (overlay), not within the emulator.

The external source is likely the reason simple graphics is broken.

**Comment by FastHit** [comment]
> NO DC is coming from an external source (overlay), not within the emulator, its from GFE.
> 
> GFE is likely the reason simple graphics is broken.

1. what is GFE?
2. how can i fix it?

**Comment by Squall-Leonhart** [comment]
> what is GFE?

Geforce Experience

> how can i fix it?

Disable the Geforce Experience overlays.

**Comment by FastHit** [comment]
> > what is GFE?
> 
> Geforce Experience
> 
> > how can i fix it?
> 
> Disable the Geforce Experience overlays.

i disabled it and it still shows it, i tried to also disable the xbox game bar(the one that built in win10) one, and it still shows it.

**Comment by FastHit** [comment]
> Please use the template instead of substituting your own. We need to know what platform (which I know you mentioned already), what Version of the emulator you're using etc the small details in that issue template helps us diagnose your issue better.

i editted it to have the template.

**Comment by Squall-Leonhart** [comment]
https://help.yoyogames.com/hc/en-us/articles/360019880557-Nvidia-GPU-users-might-see-No-DC-written-in-corner-of-GMS2-or-games

**Comment by FastHit** [comment]
> https://help.yoyogames.com/hc/en-us/articles/360019880557-Nvidia-GPU-users-might-see-No-DC-written-in-corner-of-GMS2-or-games

Thanks!

**Comment by Squall-Leonhart** [comment]
The cause of this has been identified, RenPy based software is causing the value to change.
